include/common-types.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECT_SDK_COMMON_TYPES_H
+#define CONNECT_SDK_COMMON_TYPES_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <boost/move/unique_ptr.hpp>
+#include <boost/chrono/system_clocks.hpp>
+
+namespace prism {
+namespace connect {
+
+using std::string;
+using std::vector;
+using std::map;
+
+using boost::movelib::unique_ptr;
```

Comment:
Using in header is intentional in particular for boost types, so we don't fully qualify anywhere, but in a single place in the header.
What is the problem with using <namespace>::<template class>? It just says when encounter <template class>, look for it in <namespace>. It's controlled and explicit way of introducing names into current namespace.
What will be typedef for vector or map?